packages/p2p/src/components/floating-rate/floating-rate.jsx

```diff
@@ -15,22 +16,37 @@ const FloatingRate = ({
     error_messages,
     fiat_currency,
     local_currency,
+    onChange,
     offset,
-    placeholder,
     ...props
 }) => {
     const { general_store } = useStores();
     const { name, value, required } = props;
-    const market_feed = value ? parseFloat(exchange_rate * (1 + value / 100)).toFixed(2) : exchange_rate;
+    // Get market feed restricted to 2 decimal places
+    const market_feed = value ? truncateDecimal(parseFloat(exchange_rate * (1 + value / 100)), 2) : exchange_rate;
 
+    const onBlurHandler = e => {
+        let float_rate = e.target.value;
+        if (float_rate && float_rate.length) {
+            float_rate = parseFloat(float_rate).toFixed(2);
+            if (!/^[+-]/.test(float_rate) && Math.sign(float_rate) > 0) {
```

Comment:
Why do we use `Math.sign()` here? Does it change positivity of `float_rate`?

Reply:
There is a requirement to display `+` for positive integers. Since this is not available out of the box, I am using Math.sign() to tell me whether the number is positive or negative

Reply:
I mean on line 31, you already convert float_rate to a number and on the next line you can simply check if `float_rate` is bigger than 0, right? Why do we need to use `Math.sign` here?

Reply:
Will remove math.sign